www/scripts/codecheckerviewer/ListOfRuns.js

```diff
@@ -15,9 +15,10 @@ define([
   'dijit/form/TextBox',
   'dijit/layout/BorderContainer',
   'dijit/layout/ContentPane',
-  'dojox/grid/DataGrid'],
+  'dojox/grid/DataGrid',
```

Comment:
We should keep an import order in the JS code too: `dijit/form/...`, `dijit/grid/...`, `dijit/layout/...`.

Don't forget to align the variable names in the function decl below!